src/app/facility/utility-data/energy-consumption/utility-meter-data/edit-bill/edit-utility-bill/edit-utility-bill.component.html

```diff
@@ -43,17 +43,17 @@
       <label for="totalEnergyUse">Total Energy Use <span class="required">*</span></label>
       <div class="input-group mb-3">
         <input id="totalEnergyUse" class="form-control" type="number" formControlName="totalEnergyUse"
-          onfocus="this.select();" min="0" max="999999999999999" (input)="setTotalEmissions()">
+          onfocus="this.select();" max="999999999999999" (input)="setTotalEmissions()">
         <span class="input-group-text" [innerHTML]="energyUnit | settingsLabel"></span>
       </div>
       <div class="alert alert-danger" *ngIf="meterDataForm.get('totalEnergyUse').hasError('required')">Total energy use
         is
         required</div>
-      <div class="alert alert-danger" *ngIf="meterDataForm.get('totalEnergyUse').hasError('min')">Can't be less than 0
+      <!-- <div class="alert alert-danger" *ngIf="meterDataForm.get('totalEnergyUse').hasError('min')">Can't be less than 0
```

Comment:
I think we will still want to show the error message when appropriate for min.

If the validation can never be triggered to show this message than lets delete the commented block

Reply:
Tested. Can just remove the commented out block